config/ompi_check_ucx.m4

```diff
@@ -64,7 +63,8 @@ AC_DEFUN([OMPI_CHECK_UCX],[
 	AC_MSG_CHECKING(for UCX version compatibility)
 	AC_REQUIRE_CPP
 	old_CPPFLAGS="$CPPFLAGS"
-	CPPFLAGS="$CPPFLAGS -I$ompi_check_ucx_dir/include"
+        AS_IF([test -n "$ompi_check_ucx_dir"],
```

Comment:
Shouldn't you check for `$ompi_check_ucx_dir/include`?

Reply:
I'm not sure that's the same test; this is a test to see if the string has a value, not that the directory exists...